UnitsNet.Tests/QuantityTest.cs

```diff
@@ -135,30 +135,30 @@ public void TryFrom_GivenValueAndUnit_ReturnsQuantity()
         [Fact]
         public void TryParse_GivenInvalidQuantityType_ReturnsFalseAndNullQuantity()
         {
-            Assert.False(Quantity.TryParse(typeof(DummyIQuantity), "3.0 cm", out IQuantity? parsedLength));
+            Assert.False(Quantity.TryParse(CultureInfo.InvariantCulture, typeof(DummyIQuantity), "3.0 cm", out IQuantity? parsedLength));
             Assert.Null(parsedLength);
         }
 
         [Fact]
         public void TryParse_GivenInvalidString_ReturnsFalseAndNullQuantity()
         {
-            Assert.False(Quantity.TryParse(typeof(Length), "x cm", out IQuantity? parsedLength));
+            Assert.False(Quantity.TryParse(CultureInfo.InvariantCulture, typeof(Length), "x cm", out IQuantity? parsedLength));
             Assert.Null(parsedLength);
 
-            Assert.False(Quantity.TryParse(typeof(Mass), "xt", out IQuantity? parsedMass));
+            Assert.False(Quantity.TryParse(CultureInfo.InvariantCulture, typeof(Mass), "xt", out IQuantity? parsedMass));
             Assert.Null(parsedMass);
 
-            Assert.False(Quantity.TryParse(typeof(Pressure), "foo", out IQuantity? parsedPressure));
+            Assert.False(Quantity.TryParse(CultureInfo.InvariantCulture, typeof(Pressure), "foo", out IQuantity? parsedPressure));
```

Comment:
You can use static imports to shorten this to just `InvariantCulture`

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#static-modifier